src/main/java/com/unityTest/testrunner/service/CodeService.java

```diff
@@ -183,4 +199,26 @@ private String buildPythonTestFunction(String funcName, String body) {
 		// Create code block by indenting the body by one tab
 		return def.concat(Utils.indent(body, 1).concat("\n"));
 	}
+
+	private String buildHUnitTestFunction(String funcName, String body) {
+		StringBuilder resultBuilder = new StringBuilder();
+
+		// Importing Hunit
+		resultBuilder.append("import HUnit");
+
+		// Main function
+		resultBuilder.append("\n\nmain :: IO ()\n");

```

Comment:
Not crazy about the `resultBuilder.append` everywhere. Maybe just "..." + "..." might work just fine.

Reply:
I believe the benefit over using `+` is that `+` creates a new `String` object, so I believe this is more efficient.

Reply:
Oh I see, could you do a longer `String.format()` if possible? At least then we can condense the lines a little.

Reply:
I can do some `String.format()` but I would like to keep some things in different lines to show that they are in different in the final generation. Just a personal preference.